### DISCUSSION TITLE
Adding NIST audit info

### DISCUSSION THREAD

**Discussion by dark668** [prowler-cloud/prowler#1235] [Ideas]
Is there any possibility to adding NIST compliance profile based on AWS Config conformance packs?

Examples of the checks are listed here: https://github.com/awslabs/aws-config-rules/blob/master/aws-config-conformance-packs/Operational-Best-Practices-for-NIST-800-53-rev-4.yaml

**Comment by jfagoagas** [comment]
Hi @dark668, thank you for reaching out!

Are you referring to create a new group for NIST compliance? Like the PCI one here https://github.com/prowler-cloud/prowler/blob/master/groups/group15_pci

**Reply by dark668** [reply]
Yes

**Comment by toniblyx** [comment]
This is great @dark668, adding support for NIST 800-53 rev 4 would be amazing. We would need to map all those requirements to Prowler checks. Could you help us with that?
